docs.html

```diff
@@ -87,37 +259,111 @@ <h6>Text Badge</h6>
             <img class="lg-avatar avatar" src="https://boredhumans.b-cdn.net/faces2/738.jpg" alt="user" />
             <span class="text-badge">Popular</span>
         </div>
+<pre>
+<code class="html">
+
+    <div class="m1 badge-container">
+        <img class="lg-avatar avatar" src="//boredhumans.b-cdn.net/faces2/738.jpg" alt="user" />
+        <span class="text-badge">NEW</span>
     </div>
 
+</code>
+</pre>
+<hr>
+
+    </section>
     <!-- Button Component -->
-    <h4 class="p2-left">Buttons</h4>
-    <div class="container">
-        <h6>Primary Buttons</h6>
-        <button class="m1 btn primary-btn">Primary Button</button>
-        <button class="m1 btn primary-outline-btn">Primary Outline</button>
-        <h6>Secondary Buttons</h6>
-        <button class="m1 btn secondary-btn">Secondary Button</button>
-        <button class="m1 btn secondary-outline-btn">Sec Outline Button</button>
-        <h6>Link Button</h6>
-        <button class="m1 btn link-btn">Link Button</button>
-        <button class="m1 btn underline-link-btn">Link Button</button>
-        <h6>Icon Button</h6>
-        <button class="m1 btn icon-btn"><i class="fas fa-shopping-cart"></i></button>
-        <button class="m1 btn icon-inside-btn"><i class="p1-right fas fa-shopping-cart"></i>Add to cart</button>
-        <h6>Floating Button</h6>
-        <button class="m1 btn floating-btn"><i class="fas fa-plus"></i></button>
-        <h6>Disabled Button</h6>
-        <button class="m1 btn primary-btn disabled-btn">Primary Button</button>
-        <button class="m1 btn underline-link-btn disabled-btn">Link Button</button>
-        <button class="m1 btn icon-btn disabled-btn"><i class="fas fa-shopping-cart"></i></button>
-    </div>
+    <section id="Button">
+    <h2 class="component-heading">Buttons</h2>
+    <p class="component-desc">Buttons are also called as call to action. We have range of buttons . You need to add respective classes, and you are good to go.</p>
+    <h5 class="component-subheading">Solid Buttons</h5>
+    <button class="m1 btn primary-btn">Primary Button</button>
+    <button class="m1 btn secondary-btn">Secondary Button</button>
+<pre>
+<code class="html">
 
-    <!-- Card Components -->
-    <h4 class="p2-left">Cards</h4>
+    <button class="btn primary-btn">Primary Button</button>
+    <button class="btn secondary-btn">Secondary Button</button>
+
+</code>
+</pre>
+<hr>
+    <h5 class="component-subheading">Outlined Buttons</h5>
+    <button class="m1 btn primary-outline-btn">Primary Outline</button>
+    <button class="m1 btn secondary-outline-btn">Sec Outline Button</button>
```

Comment:
Not sure if it's too late - do you think it's better if we have classes like `btn btn-secondary-outline`? Like keeping the modifiers in the end? 
Just a suggestion so that we get the mix of both BEM and OOCSS.

Reply:
Yes, thanks for the suggestion. I have made the changes.